openwisp_network_topology/api/views.py

```diff
@@ -39,46 +48,58 @@ class RequireAuthentication(APIView):
         ]
 
 
-class NetworkCollectionView(generics.ListAPIView, RequireAuthentication):
+class ListViewPagination(pagination.PageNumberPagination):
+    page_size = 10
+    page_size_query_param = 'page_size'
+    max_page_size = 100
+
+
+class NetworkCollectionView(
+    RequireAuthentication,
+    FilterByOrganizationManaged,
+    generics.ListCreateAPIView,
+):
     """
     Data of all the topologies returned
-    in NetJSON NetworkCollection format
+    in NetJSON NetworkCollection format.
     """
 
     serializer_class = NetworkGraphSerializer
-    queryset = Topology.objects.filter(published=True)
+    queryset = Topology.objects.select_related('organization')
+    filter_backends = [DjangoFilterBackend]
+    filterset_fields = ['published']
```

Comment:
shouldn't a filter appear in the browsable API? I don't see any filter there.
Also, is this being documented? It doesn't look like it.
So at the moment this new feature is not discoverable nor documented, which will most likely mean nobody will even know it exists.
To maintain a bit of backward compatibility here, can we make sure that by default only published items are shown and then the user of the API can ovverride this behavior if needed?

Reply:
`include_unpublished` is not implemented as a true filter using `django-filters`. Therefore, it does not appear in browsable API view.